src/app/views/main-header/settings/ThemeChooserV9.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect } from 'react';
+import { makeStyles, RadioGroup, Radio} from '@fluentui/react-components';
+import { useAppDispatch, useAppSelector } from '../../../../store';
+import { componentNames, eventTypes, telemetry } from '../../../../telemetry';
+import { PopupsComponent } from '../../../services/context/popups-context';
+import { changeTheme } from '../../../services/slices/theme.slice';
+import { loadGETheme } from '../../../../themes';
+import { translateMessage } from '../../../utils/translate-messages';
+import { BrightnessHighRegular, WeatherMoonFilled, CircleHalfFillFilled } from '@fluentui/react-icons';
+
+const availableThemes = [
```

Comment:
Since you've added system theme support, make it an option and probably the default selected.

Reply:
It's the default selected ln70-75

Reply:
I meant on the popup 
![image](https://github.com/user-attachments/assets/93eea600-0dc4-4469-8f50-3db6f5f287e4)